### PR TITLE
Packaging section and FPM example

### PR DESCRIPTION
**Pull request by jmdf**
I've been using a singularity container with [FPM](https://github.com/jordansissel/fpm) installed to package everything I need, across different systems/environments

It works pretty well and I think it's a good example to have here.

### CODE REVIEW

File: packaging/fpm/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+To build this container, you will need the following:
+* Singularity installed
+* Internet access
+* Root access
+
+## Build instructions
+
+To build the container, use:
+
+`sudo singularity build fpm.simg fpm.def`
```
Comment (GodloveD) [inline packaging/fpm/README.md]: can you change `fpm.simg` to `fpm.sif` throughout to reflect the new SIF format in 3.0?

**Review by GodloveD — CHANGES_REQUESTED** [review]
Heya @jmdf !  Thanks so much for the contribution.  Sorry it took a little time to see and review.  Can you add this directory and file to the top level README.md so that it will be reflected in the map there?  Other than that I'm and the one other comment about simg to sif I'm happy to approve and merge.

**Comment by jmdf** [comment]
Hey @GodloveD !
Files changed as requested and it's ready to be reviewed again.

Thanks for the input.

**Comment by GodloveD** [comment]
Thanks @jmdf!  Looks good!

**Review by GodloveD — APPROVED** [review]

**Pull request merged**